mne/io/eeglab/eeglab.py

```diff
@@ -390,6 +390,34 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
                 info, data, filenames=[input_fname], last_samps=last_samps,
                 orig_format='double', verbose=verbose)
 
+        # create event_ch from annotations
+        annot = read_annotations_eeglab(input_fname)
+        self.set_annotations(annot)
+
+        dropped_desc = []  # use to collect dropped descriptions
+        event_id_ = partial(_event_id_func,
+                            event_id=event_id,
+                            event_id_func=event_id_func,
+                            dropped=dropped_desc)
+        events, _ = events_from_annotations(self, event_id=event_id_)
+        if not events.size:
+            logger.info('No events found, returning empty stim channel ...')
+            warn(' As is, the trigger channel will consist entirely of zeros.',
+                 RuntimeWarning)
```

Comment:
you stick the old messages? in old behavior?

Reply:
the io should not change, should it?

Reply:
No need for `RuntimeWarning` here, it's the default class